code/__DEFINES/obj_flags.dm

```diff
@@ -95,6 +95,8 @@
 #define HEADINTERNALS (1<<16)
 /// Prevents masks from getting adjusted from enabling internals
 #define INTERNALS_ADJUST_EXEMPT (1<<17)
+/// Determines if our piece of clothing is carpy or martial arty Sleeping Carp's style factor.
+#define CARP_STYLE_FACTOR (1<<18)
```

Comment:
Sorry, what kind of code documentation is that?! What does it that even mean or entail? lmao.

Reply:
I wrote this while half asleep, give me a break 📦